.github/workflows/extension-issue-handling.yml

```diff
@@ -1,13 +1,10 @@
 name: Extension Issue Handling
 on:
   issues:
-    types: [opened, labeled]
+    types: [opened]
 
 jobs:
   handle-extension-issues:
```

Comment:
you need to add permissions

Reply:
> you need to add permissions

Plz add mne abhi abhi laptop bandh Kiya he

Reply:
Keep the types to open, cause it runs twice